.github/workflows/mk-docs-build.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
I believe it's best to pin an ubuntu version here, stick with 18.04

Reply:
K I will Pin everything to Ubuntu 18.04 for now. We can bump it later.